src/VisualStudio/Core/Def/Implementation/VsRefactorNotifyService.cs

```diff
@@ -101,7 +101,9 @@ public bool TryOnAfterGlobalSymbolRenamed(Workspace workspace, IEnumerable<Docum
 
             hierarchyToItemIDsMap = null;
             rqnames = null;
+#pragma warning disable CA2000 // Dispose objects before losing scope
             if (!TryGetItemIDsAndRQName(workspace, changedDocumentIDs, symbol, out var visualStudioWorkspace, out hierarchyToItemIDsMap, out var rqname))
+#pragma warning restore CA2000 // Dispose objects before losing scope
             {
```

Comment:
Please don't add an trivia between the `if` and opening `{`. I'd move it after `{`